components/users.py

```diff
@@ -37,11 +46,11 @@ def add(self, user: User):
 
         self.save()
 
-    def get(self, id: int):
-        for user in self.users:
-            if user.id == id:
-                return user
-        return None
+    def get(self, user: discord.User) -> User:
```

Comment:
Not sure why we're bothering with this.  Fetching by ID was working fine, and we're not getting any value in sending down the entire object.